css/main.css

```diff
@@ -14,50 +14,53 @@
     flex-direction: column;
     width: 100vw;
     height: 100vh;
-  }
-
-  #header nav button {
     position: fixed;
-    left: 24px;
-    top: 10px;
-    border: none;
-    background: transparent;
-    font-size: 44px;
-    cursor: pointer;
+    background: #fff;
+    transition: all 0.3s ease;
```

Comment:
- [ ] As said by the previous reviewer kindly consider fixing the partner section as you can see it's very small and the elements are not well organized;
- [ ] You may consider also fixing the image path on the last sponsor as you can how it's broken
- [ ] Kindly make sure also the sponsor images are not overlapping as it is the case on your desktop version and the event section must be visible.

![image](https://user-images.githubusercontent.com/97100091/196746087-aa9f5928-487e-4c71-b830-8435355ecc23.png)

![image](https://user-images.githubusercontent.com/97100091/196746400-cabb28a0-b33c-41f8-9408-77b95de926c9.png)